selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/Button.java

```diff
@@ -38,9 +38,9 @@
 
   public final By ELEMENT_SAVE_CLOSE_BUTTON_2 = By.xpath("//*[text()='Save And Close']");
 
-  public final By ELEMENT_OK_BUTTON = By.xpath("//*[text()='OK']");
+  public static final By ELEMENT_OK_BUTTON = By.xpath("//*[text()='OK']");
```

Comment:
This class contains generic buttons that we can use in some cases.There are not used in all cases,It was created in the old project .